<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyerChecksDialog</class>
 <widget class="QDialog" name="BuyerChecksDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>521</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Print Buyer Checks</string>
  </property>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Q3ListBox" name="buyerList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>510</width>
     <height>450</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans Mono</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>Q3ListBox::Multi</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="reportCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print A Report With The Check</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/commissiondao.h</include>
  <include location="local">types/invoice.h</include>
  <include location="local">dao/invoicedao.h</include>
  <include location="local">dao/checkdao.h</include>
  <include location="local">dao/setupdao.h</include>
  <include location="local">reports/textcheck.h</include>
  <include location="local">dao/printerdao.h</include>
  <include location="local">reports/buyercheckreport.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
